pages/common/npx.md

```diff
@@ -0,0 +1,15 @@
+# npx
+
+> Execute binaries from `npm` packages(most of the cli tools) .
+
+- Execute the npm packages binaries(Package will be installed if it does not exist in node_modules or $PATH):
```

Comment:
I thought `Execute the installed npm module.` sounded better. But if you want to keep your style, we can do `Execute the binary from a given npm module`. I have used module rather than package to match with the example below since you have used "module_name".

Also, give a space before "(". 

And I intentionally did not mention the node_modules or $PATH locations for simplicity here. Essentially, we like to keep things at a high level and not bother to go into details unless they are required to understand the command. I feel that just informing the user that the package will be installed if it is not, is good enough. There is not need to explain how it detects whether a package is installed or not. However, if you think that it is important, you can keep it.

Reply:
ha, I make a mistake, want explain specific thing however it will more  details.